ios/Podfile.lock

```diff
@@ -696,4 +696,4 @@ SPEC CHECKSUMS:
 
 PODFILE CHECKSUM: 41b806c7f131f87b716be1f1f9377532d6c9e43a
 
-COCOAPODS: 1.10.0
+COCOAPODS: 1.10.0
```

Comment:
I think maybe this change not intentional?

Reply:
Yes, sorry about that. I reverted it back to a previous version.